### PR TITLE
Fixed some changes to the tag_iam_user

### PR DESCRIPTION
**Pull request by athiruma**
Create the spreadsheet, if it is not present, and add the user's list

### CODE REVIEW

File: cloud_governance/policy/policy_operations/aws/tag_user/tag_iam_user.py [hunk 1]
```diff
@@ -89,7 +91,7 @@ def generate_user_csv(self):
         tag_values = {}
         for user in users:
             user_name = user.get('UserName')
-            if '-' not in user_name:
+            if user_name.count('-') <= 3:
```
Comment (ebattat) [inline cloud_governance/policy/policy_operations/aws/tag_user/tag_iam_user.py:94]: can we based on count '-' number ?

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/tag_user/tag_iam_user.py:94]: Not actually, but we need to do it now. As we are moving to multiple accounts.

Comment (ebattat) [reply cloud_governance/policy/policy_operations/aws/tag_user/tag_iam_user.py:94]: Are you sure it will not affect backward compatibility ?

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/tag_user/tag_iam_user.py:94]: yes, it won't

**Review by ebattat — APPROVED** [review]
/lgtm

**Pull request merged**